docs/content/contributing/guidelines-frontend.en-us.md

```diff
@@ -65,14 +65,17 @@ Recommended implementations:
 
 * Vue + Vanilla JS
 * Fomantic-UI (jQuery)
+* htmx
 * Vanilla JS
 
 Discouraged implementations:
 
 * Vue + Fomantic-UI (jQuery)
 * jQuery + Vanilla JS
+* htmx + anything else
```

Comment:
```suggestion
* htmx + other framework which requires heavy JS code, or unnecessary features like htmx script
```

Reply:
what's htmx scripting?

Reply:
Write a lot of JS code inline .... https://htmx.org/docs/#scripting

Actually such inline code could be done by VanillaJS or jQuery in most cases.

![image](https://github.com/go-gitea/gitea/assets/2114189/0f83fe39-402e-40c9-a897-f1819c601d1b)

Reply:
So we're against the use of `hx-on`?

Reply:
I guess so. Before, the inline script (eg: `onclick`) has been forbidden. So I do not think enough people would like to use `hx-on` here.

The problem is: https://github.com/go-gitea/gitea/issues/28880#issuecomment-1902643849 :

> Some advanced features like "htmx script" is quite fragile and I don't know how to lint/test them. It would cause maintainability problems.

Reply:
Dpne